QSB/QSBNetworkManager.cs

```diff
@@ -67,8 +88,11 @@ private void ConfigureNetworkManager()
             customConfig = true;
             connectionConfig.AddChannel(QosType.Reliable);
             connectionConfig.AddChannel(QosType.Unreliable);
+            this.SetValue("m_MaxBufferedPackets", MaxBufferedPackets);
```

Comment:
Probably not needed? But with the amount of network messages we send, might be a good idea